data/human/free worlds 3 reconciliation.txt

```diff
@@ -1326,11 +1326,44 @@ mission "FW Pug 2C: Jump Drive"
 
 
 
+mission "FW Pug 2C: Plugin Drive"
+	landing
+	name "Investigate the Pug"
+	description "Travel to the Deneb system to investigate the Pug; if possible, make contact with them and determine what their goals and desires are."
+	source Hephaestus
+	destination Pugglemug
+	clearance
+	to offer
+		has "FW Pug 2B: done"
+		not "fw given jump drive"
+		not "flagship attribute: jump drive"
+		not "outfit (flagship installed): Hyperdrive"
+		not "outfit (flagship installed): Scram Drive"
+	passengers 1
+	on fail
+		dialog `You have failed an essential Free Worlds mission. If you want to complete the story line, revert to the autosave or another earlier snapshot of the game.`
+	
+	on offer
+		conversation
+			`You are directed to land in a hangar some distance away from the spaceport, in a cluster of buildings owned by Syndicated Systems. A team of engineers meets you and quickly and efficiently installs a strange, alien-looking device that you assume must be a jump drive. "Now, be careful with your fuel gauge," one of them tells you. "This thing guzzles twice as much fuel per jump as an ordinary hyperdrive, and you don't want to become stranded while outside human space."`
+				accept
+	
+	on accept
+		outfit "Jump Drive" 1
+		set "fw given jump drive"
+	on visit
+		dialog `You land on <planet>, but realize that your escort carrying Alondo hasn't entered the system yet. Better depart and wait for it to arrive.`
+	on complete
+		set "FW Pug 2C: done"
+
+
+
```

Comment:
I don't think we want to be maintaining stuff to keep plugin compatibility with the story. There's nothing stopping a plugin that adds an exotic drive from adding a mission like this to handle its situation.

Reply:
This line is already present in every mission and would be necessary if you used a Shooting Star.

Reply:
Sorry about the comments, I was confused about what this PR was doing.